index.html

```diff
@@ -0,0 +1,30 @@
+<!--HTML layout page-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css" type="text/css">
+    <title>Portfolio | Sully</title>
+</head>
+<body class="main-content">
+    <header class="mainheader">
+        <h2 class="usernamehandle">bettercallsully</h2>
+        <nav class="toolbar">
+            <img class="hamburger" src="myassets/hamburger.png" alt="Hamburger Logo">
+        </nav>
+    </header>
+    <article>
+         <section class="articletext">
+             <h2 class="introduction">I'm Sully<br>Glad to see you!</h2>
+             <h3 class="selfdescription">I'm a software developer! I can help you build a product, feature or website. Look<br>Through some of my work and experience! If you like what you see and have a<br>project you need coded, don't hesitate to contact me</h3>
+             <h4 class="contactme">LET'S CONNECT</h4>

```

Comment:
- [ ] Let's make this element a link and attach a `<a>` tag or convert it to a `<button>` tag. It makes more sense when the text that actually referring to a link is to be a link such as "Let's connect" and "See project", or "Get my resume".